articles/deploying-nodejs-web-app/index.md

```diff
@@ -0,0 +1,143 @@
+
+The first step is to install Nginx. Type the following to update your package list (so you can get the latest version) and install Nginx:
+
+```bash
+sudo apt-get update
+sudo apt-get install nginx
+```
+
+Since DigitalOcean droplets are created with a firewall enabled, you’ll have to allow Nginx through it so it can work properly.  `sudo ufw allow 'Nginx Full'` will do this.
+
+To check the installation has gone smoothly, go to the http version of your domain name e.g. http://yourdomain.com. If you see a Welcome to Nginx landing page then it’s been successful.
+
+The second step is to secure your reverse proxy. Currently going to https://yourdomain.com won’t work. That’s because we haven’t configured SSL yet and we need to install a package called Certbot to do so.
+
+To install Certbot, type the following to ensure you get the latest version:
+
+```bash
+sudo add-apt-repository ppa:certbot/certbot
+sudo apt-get install python-certbot-nginx
+```
+
+Next, you need to add your domain to Nginx so Certbot can generate a certificate to the correct domain. Open the configuration file using `sudo nano /etc/nginx/sites-available/default` and replace the underscores in the server_name line to your domain. For example  `server_name yourdomain.com www.yourdomain.com;`. Save the file and exit by typing CTRL+X, y and then enter.
+
+To test, there are no errors in the file, type  `sudo nginx -t`  and if there’s none type  `sudo systemctl reload nginx`  to reload Nginx so it will use the updated configuration.
+
+Now we just need to generate the SSL certificate.  `sudo certbot --nginx -d yourdomain.com -d www.yourdomain.com` will start the process.  You should choose option 2 for the redirect process because it will forward anyone trying to access the insecure version of your site (http) to the secure (https) version instead.
+
+To test this, go to https://yourdomain.com and you should see the Nginx Welcome screen again.
+
+Finally, we're onto the last step, adding the Nginx configuration for your web app. Since it's easiest, we'll just modify the default one instead of creating a new one specifically for your web app. If you need to host several web apps on one droplet, you'd need to add a new configuration for each site.
+
+Type: `sudo nano /etc/nginx/sites-available/nano/default` to edit the default configuration file.
+
+You need to change the server_name parameter to the name of your domain. For example: yourdomain.com. Under location /, proxy_pass should be changed to http://localhost:(port name). The ssl_certificate_key should be modified: `/etc/letsencrypt/live/(domain name)/privkey.pem`. Finally, add the code block below to the end of the file and then type CTRL+X and then y to exit.
+
+```
+bash
+server {
+    if ($host = auroraspotter.space) {
+        return 301 https://$host$request_uri;
+    } # managed by Certbot
+        listen 80 default_server;
+        listen [::]:80 default_server;
+        server_name auroraspotter.space;
```

Comment:
Up until this point you have been using yourdomain.com and whatnot. Keep it consistent instead of switching to auroraspotter.space. This will confuse readers.